### ISSUE TITLE
club=scout has lower priority than the named building=civic it is hosted in

### ISSUE THREAD

**Issue by smopucilowski** [gravitystorm/openstreetmap-carto#4199]
### Expected behavior
A local scout group is based inside a named hall. At all z-levels displayed name of the civic building overrides displaying the scout group.

### Actual behavior
I imagine that the scouting group at the civic building would have higher rendering priority.

### Links and screenshots illustrating the problem

node 7855744156
way 494559020
![image](https://user-images.githubusercontent.com/5922263/91952222-33bffa80-ed44-11ea-8f27-5a88daf8a19b.png)

**Comment by polarbearing** [comment]
No values of the club=* tag are currently rendered in this style. 

As for your tagging, you have tagged both the building and a node inside with that club tag, which seems redundant.
If the club uses the whole building, I would recommend to remove the node, use the building polygon only, and add
```amenity=community_centre``` with a suitable sub-tagging ```community_centre=*``` for its type and ```community_centre:for=*``` for the target group. E.g. community_centre=club_home and community_centre:for=scout .

**Comment by jeisenbe** [comment]
I've changed the title of this issue to make it a request to "render club=scout". 

Surprisingly, we do not have an issue open about rendering features under the key "club=". 

The most common value is "club=sport" with slightly over 10,000 uses. Only have have another main feature key, mainly `building=` plus about 11% with `leisure=`.

The generic value `club=yes` is next with under 5000 uses - we may not want to render this, since a more specific value would be better. 

Then `club=scout` is third, with over 2000 uses - so not terribly common. None of the other values have over 1000 uses.

The distribution is mostly limited to part of Europe and southern South America - there are very few in North or central America, Asia or Africa, though it's used in Australia and New Zealand:
<img width="697" alt="Screen Shot 2020-09-05 at 23 52 36" src="https://user-images.githubusercontent.com/42757252/92320005-e2796a80-efd2-11ea-82f8-28daeb784717.png">

**Comment by jeisenbe** [comment]
There was previously some discussion about this key in #3611 and PR #4020 where I considered it unlikely to be rendered.

An issue is that a club is a group of people or an organisation, rather than a physical feature. Some types of clubs can meet in different locations each week. 

The physical location where clubs meet might be mapped with another tag such as `amenity=social_centre` as mentioned on the [Key:club wiki page](https://wiki.openstreetmap.org/wiki/Key:club): "The base of the club can be tagged amenity=community_centre with the type community_centre=club_home and possibly the target group community_centre:for=*; either on the containing building or the same element as the club itself."

**Comment by polarbearing** [comment]
Indeed, club=* is a non-physical tag, thus it serves as a sub-tag for further specification of different amenity=community_centre.
Unless there is an idea to render the base of a sports club home different from a base of a scouts club, I see no current application of this key in this style.

**Comment by polarbearing** [comment]
Indeed, club=* is a non-physical tag, thus it serves as a sub-tag for further specification of different amenity=community_centre.
Unless there is an idea to render the base of a sports club home different from a base of a scouts club, I see no current application of this key in this style.

**Comment by polarbearing** [comment]
Indeed, club=* is a non-physical tag, thus it serves as a sub-tag for further specification of different amenity=community_centre.
Unless there is an idea to render the base of a sports club home different from a base of a scouts club, I see no current application of this key in this style.

**Comment by jeisenbe** [comment]
> it serves as a sub-tag for further specification of different amenity=community_centre.

I originally suspected that, but taginfo does not confirm: https://taginfo.openstreetmap.org/keys/club#combinations - only 10% are combined with `amenity=`.

**Comment by Zalitoar** [comment]
About the physical location of clubs of any kind, I don't know how they work in Europe but at least in Latin America there are very well stablished in a physical location, mostly of them permanently. That's the case of Scout and Guide groups because the have frequently a common open space for outdoor activities and offices for kitchen, camping elements, and other stuff like wooden constructions, tools, etc.  So, they're usually working on big places that own by themselves or are owned by sponsoring organizations such as churches, schools, military facilities and others.

They belongs to different national or international associations but considered a 'movement' as a whole. 
If there last any doubt on how much groups would be out there, we can have in mind that according to the United Nations the Scout/Guide movement is one of largest youth movement in the world, with over 60 million active members and present in 216 countries and territories, so they are a lot of groups not being mapped yet.

In Argentina only I mapped more than 850 groups and there are few more waiting to be added. You can check it out with [Overpass-Turbo](http://overpass-turbo.eu/s/XSs) the amount of groups and guess how much left to map in the rest of the world.

I suggest to keep this issue open because as said above the number of nodes for this kind of element will eventually raise.

**Comment by Adamant36** [comment]
> I suggest to keep this issue open because as I said above the number of nodes for this kind of element will eventually raise. Also it would be added to the render with a specific symbol for clubs or Scout / Guide group if the number of them increase many times the actual one.

Likely some of the lack of usage has to do with the term "scout" being a western/English one that not all "scout" organizations in none western English countries use. I was just reading on Wikipedia about an organization from somewhere (I can't remember where now though) that was essentially a "scouting" group, but they didn't call themselves that. So, more then likely those places will never be tagged as club=scout. Since it's just not a term that they refer to themselves as. Also, I'd guess some of the club=youth and club=social uses are really better tagged as club=scout. So that might be something to look into. Perhaps finding a more universal term then "scout" would be to.

**Comment by Marc-marc-marc** [comment]
> An issue is that a club is a group of people or an organisation, rather than a physical feature

how is this different from office=association office=company leisure=hackerspace ?

> Some types of clubs can meet in different locations each week.

associations too, but this does not prevent both from having an official fixed place visible on the ground. club=* does not indicate a group of people but something that persists even when all the people in the club have left the premises (the name on the premises, any trophies, painting, photo, administrative items, tables and chairs and any other equipment needed by the club remain, in the same way that the offices of a company remain even when the employees have left.

**Comment by Adamant36** [comment]
With things like the Cub Scouts they are kind of more transient as a aren't they?  Like, there's a usually a central local office where they do paperwork and crap, but they don't do actual club activities there? Wheres, with something like A.A. or similar "clubs" the office is usually also where they have weekly meetings. At least that's how I understand it. 

There's also a lot of overlap between things like this and outreach social facilities. To the point where they are to similar to make rendering this helpful IMO.

**Comment by Zalitoar** [comment]
> I was just reading on Wikipedia about an organization from somewhere (I can't remember where now though) that was essentially a "scouting" group, but they didn't call themselves that. So, more then likely those places will never be tagged as club=scout. Since it's just not a term that they refer to themselves as. Also, I'd guess some of the club=youth and club=social uses are really better tagged as club=scout. So that might be something to look into. Perhaps finding a more universal term then "scout" would be to. Some could also possibly be tagged as social facilities.

I agree with you about that similar associations  do not use the name 'Scout', that's the case of the Guide movement, and others around the world. All of them are mainly focused in youth non-formal education but they have also members of all ages developing activities. 

The largest and more numerous by far is the Scout movement, perhaps that's the reason why is used as value for the club tag.

In Wikipedia the article 'Scouting' defines it as: 

> The Scout movement, also known as Scouting or the Scouts, is a voluntary non-political educational movement for young people.

Therefore this and other similar movements or associations would fit under club=youth classification as @Adamant36 proposed. I would add other tag for specify the movement or association of the club.

Scout / Guide groups (that is the denomination they use) meet at least once a week in the same place that could be an office, building, field or even a camping they usually own, so they aren't temporal meetings in random locations. Most of the groups were founded many decades ago so they're very known in their neighbourhoods and citiies.
